sklearn/utils/_param_validation.py

```diff
@@ -506,6 +531,42 @@ def __str__(self):
         )
 
 
+class _MissingValues(_Constraint):
+    """Helper constraint for the `missing_values` parameters.
+
+    Convenience for
+    [Integral, Real, str, np.nan, None, pd.NA]
```

Comment:
I prefer to write expactly what would be the constraint if we'd put it in a _parameter_constraints dict
```suggestion
    [Integral, Real, str, None, _NanConstraint, _NAConstrint]
```